docs/video/01-build/08-Develop-Managing-Media-Stream.mdx

```diff
@@ -20,10 +20,10 @@ import { ABlock } from "@site/src/css/SharedStyling";
 
 ## Enable / Disable Local Media
 
-To enable/disable local media, we'll call the `media()` method on the `aliceVideoCall`.
+To enable/disable local media, we'll call the `config()` method on the `aliceVideoCall`.
 
 ```typescript
-  aliceVideoCall.media({
+  aliceVideoCall.config({
     video?: Boolean,
     audio?: Boolean
```

Comment:
This section should be now Managing Media Config since we have moded to config, refer to chat to see how API and parameters are segregated